deploy-tool/main.go

```diff
@@ -185,7 +190,20 @@ func generateAndDeployKeyPair(namespace, keyName string, keyMarshaler privateKey
 	return p256Key, nil
 }
 
+func manifestExists(fqdn, dsp string) bool {
+	// Remove the locality name from the FQDN
+	path := fmt.Sprintf("https://%s/%s-manifest.json", fqdn, dsp)
+
+	resp, err := http.Get(path)
+	if err != nil {
+		log.Printf("error when getting manifest %s: %s", path, err)
```

Comment:
I don't think we need to log here since this isn't really an error condition. Could this just `return false`?

Reply:
Hmm, non 200s don't return an error. This error means something actually broke. I'll actually convert it to a fatalf since the operator probably doesn't want it to continue messing up the environment.